CrossMod/FilteringOption.cs

```diff
@@ -45,18 +42,14 @@ public abstract partial class FilteringOption : ModTexturedType, ILocalizedModTy
 			ModTypeLookup<FilteringOption>.Register(this);
 
 			Type = FilteringOptionLoader.Add(this);
+
+			Tooltip = Language.GetOrRegister(Mod, $"FilteringOption.{Name}");
```

Comment:
this code should not be here.  please use the existing code in MagicStorage/1.4.4 as a comparison